.github/workflows/regression-tests.yml

```diff
@@ -0,0 +1,91 @@
+name: REGRESSION TESTER - AGGLAYER
+on:
+  workflow_dispatch:
+    inputs:
+      zkevm_agglayer_commit_id:
+        description: '0xPolygon/agglayer (commit id)'
+        required: true
+      bake_time:
+        description: 'bake time (minutes)'
+        required: true
+
+jobs:
+  deploy_devnet:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.before }}
+
+    - name: Set up Docker
+      uses: docker/setup-buildx-action@v1
+      
+    - name: Install Foundry
+      uses: foundry-rs/foundry-toolchain@v1
+
+    - name: Clone internal kurtosis-cdk repo
+      run: |
+        git clone https://github.com/0xPolygon/kurtosis-cdk.git
+        cd kurtosis-cdk
+        git checkout dan/jit_containers
+
+    - name: Clone and build agglayer
```

Comment:
nit: maybe clone and build the agglayer just after setting up docker then you can install foundry, kurtosis etc.

Reply:
ack, might circle back to these particular changes in near future. appreciate the detailed review !

Reply:
ended up just cleaning up now